### DISCUSSION TITLE
Make 'enableRGWDashboard' configurable by custom？

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#11884] [Q&A]
https://github.com/rook/rook/blob/6273bc1aec7087465829195a89378abba7d5ec8a/pkg/operator/ceph/object/rgw.go#L85

if we want to disable the rgw dashboard, is there a button to control it?

**Comment by parth-gr** [comment]
It doesn't seem to right now,
If the rgw user is created it will auto-create it,

If there is a specific use case you can open an issue|
Thanks
cc @thotz

**Reply by thotz** [reply]
It is not a custom option, this will be enabled by default
